openmdao_book/features/building_blocks/solvers/linear_block_gs.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Missing the automethod for the constructor (see original doc)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>